provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/configs/DeviceRegistrationState.java

```diff
@@ -52,14 +52,16 @@
     private static final String CREATED_DATETIME_UTC_TAG = "createdDateTimeUtc";
     @Expose
     @SerializedName(CREATED_DATETIME_UTC_TAG)
-    private final String createdDateTimeUtc = null;
+    @SuppressWarnings("unused") // used by reflection during json serialization/deserialization
+    private String createdDateTimeUtc;
     private transient Date createdDateTimeUtcDate;
 
     // last update date and time
     private static final String LAST_UPDATED_DATETIME_UTC_TAG = "lastUpdatedDateTimeUtc";
     @Expose
     @SerializedName(LAST_UPDATED_DATETIME_UTC_TAG)
-    private final String lastUpdatedDateTimeUtc = null;
```

Comment:
creating a final string that will always be null doesn't make much sense, so I removed the final qualifier so that it can be json serialized/deserialized with meaningful data.